VSharp.ML.AIAgent/ml/common_model/utils.py

```diff
@@ -124,10 +123,28 @@ def load_best_models_dict(path):
             if row[1] != best_models[row[0]][2]:
                 path_to_model = os.path.join(common_models_path, row[1])
                 ref_model = load_model(
-                    Path(path_to_model), model=GeneralConfig.EXPORT_MODEL_INIT()
+                    Path(path_to_model), model=StateModelEncoderLastLayer(32, 8)
                 )
-
                 ref_model.load_state_dict(torch.load(path_to_model))
                 ref_model.to(GeneralConfig.DEVICE)
                 best_models[row[0]][0] = ref_model
+                best_models[row[0]][1] = ast.literal_eval(row[2])
```

Comment:
а, вижу. лучше переписывать значения
```suggestion
                best_models[row[0]] = (ref_model, ast.literal_eval(row[2]), row[1])
```

Reply:
поправила